docs/docusaurus.config.js

```diff
@@ -111,6 +111,10 @@ const config = {
                 label: 'Blog',
                 to: '/blog',
               }, */
+              {
+                label: 'Status',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/KMU-Dev/wm-toolkit/pullRequest?prid=8339100)
```suggestion
                label: "Status",
```